content/dart/concept/list/terms/remove/remove.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: '.remove()'
+Description: 'Removes first occurrence of a value from the list.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Dart'
+  - 'Functions'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+The **`.remove()`** is a method to remove the first occurrence of an item from the list and return bool `true` if the item is present and is successfully removed, otherwise return bool `false` if the item is not present.
```

Comment:
```suggestion
The **`.remove()`** method is used to remove the first occurrence of an item from the list and return bool `true` if the item is present and is successfully removed, otherwise, return bool `false` if the item is not present.
```